k2/python/k2/fsa.py

```diff
@@ -222,6 +222,8 @@ def __init__(
             self.__dict__[name] = dict()
 
         self._tensor_attr['scores'] = _k2.as_float(self.arcs.values()[:, -1])
+        self._tensor_attr["labels"] = self.arcs.values()[:, 2]
+        self.labels_version = self._tensor_attr["labels"]._version
```

Comment:
Could you add a comment here explaining why
we need `labels_version`?

You can place the link to this PR https://github.com/k2-fsa/k2/pull/1140
to the comment.

Reply:
ok